grails-app/domain/org/pih/warehouse/product/Product.groovy

```diff
@@ -596,15 +596,19 @@ class Product implements Comparable, Serializable {
         }
     }
 
-    def validateRequiredFields() {
+    Boolean validateRequiredFields() {
```

Comment:
you could use primitive `boolean` for that, since it never returns `null`

Reply:
makes sense, will do